libraries/ms-lib/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Defines $(LIB)_SRC, $(LIB)_DEPS, and $(LIB)_CFLAGS for the library makefile.
+# $(LIB)_SRC_ROOT, $(LIB)_INC_ROOT, and $(LIB)_OBJ_ROOT have been pre-defined.
+
+# Find all *.c files in root source and device-specific source folders.
+$(LIB)_SRC := $(wildcard $($(LIB)_SRC_ROOT)/*.c) \
+              $(wildcard $($(LIB)_SRC_ROOT)/$(DEVICE_FAMILY)/*.c)
+$(LIB)_DEPS := stm32f0xx
```

Comment:
Will this always depend on this? I feel like for ms-lib this should just pull in the library based on the device family not a static name.